.github/workflows/release-new-app.yml

```diff
@@ -0,0 +1,50 @@
+name: New release app
+
+on:
+   release:
+     types: [published]
+
+permissions:
+  contents: read
+  packages: read
+
+jobs:
+  create-archive:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
```

Comment:
**issue**: Please ensure that we check out the actual release and not the current main branch.

Reply:
what do you suggest to add here?

Reply:
You need to set a `ref` (see [docs](https://github.com/actions/checkout/tree/v4/?tab=readme-ov-file#usage)). Should also use v4 instead of v3.

Reply:
ok, I did. Thanks